magpie-run-distributed

```diff
@@ -33,46 +33,61 @@ source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-node-identification
 source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-run
 
 source ${MAGPIE_SCRIPTS_HOME}/magpie/run/magpie-run-project-tensorflow
+source ${MAGPIE_SCRIPTS_HOME}/magpie/run/magpie-run-project-tensorflow-horovod
 
 # Output some general info
 
 # Note that if MAGPIE_HOSTNAME_SCHEDULER_MAP set, MAGPIE_NODELIST may
 # have not been converted for this output.
 
-if [ "${MAGPIE_CLUSTER_NODERANK}" == "0" ]
-then
-    echo "*******************************************************"
-    echo "* Magpie General Job Info"
-    echo "*"
-    echo "* Job Nodelist: ${MAGPIE_NODELIST}"
-    echo "* Job Nodecount: ${MAGPIE_NODE_COUNT}"
-    echo "* Job Timelimit in Minutes: ${MAGPIE_TIMELIMIT_MINUTES}"
-    echo "* Job Name: ${MAGPIE_JOB_NAME}"
-    echo "* Job ID: ${MAGPIE_JOB_ID}"
-    echo "*"
-    echo "*******************************************************"
-fi
+echo "*******************************************************"
```

Comment:
why the removal of the `if [ "${MAGPIE_CLUSTER_NODERANK}" == "0" ]`?  Wouldn't it output the same thing on each rank as a result?

Reply:
You are right, `if` should be there, I will fix it.